testsuite/forge.py

```diff
@@ -320,7 +320,7 @@ def get_dashboard_link(
         raise Exception(f"Invalid refresh argument: {time_filter}")
 
     return (
-        f"{GRAFANA_BASE_URL}&var-namespace={forge_namespace}"
+        f"{GRAFANA_BASE_URL}&var-namespace={forge_namespace}&var-metrics_source=All"
```

Comment:
@rustielin - Since the namespace filter is unique that should be sufficient, we don't need to put additional constraint on metrics source. Imagine, if we start pushing the metrics from telemetry instead of vmagent, in that case otherwise, we would have to update the link again.